recipes/sagemath-db-conway-polynomials/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+curl -L -O https://raw.githubusercontent.com/sagemath/sage/7.5.1/build/pkgs/conway_polynomials/spkg-install
```

Comment:
Can you add a note here explaining why you need this to help possible future maintainers? (No need to do it now, it can be in the feedstock.)